### PR TITLE
pd: add patrol-region-worker-count

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@lhy1024 Could you please invite a tech reviewer? Thanks!

**Review by niubell — APPROVED** [review]

File: pd-configuration-file.md [hunk 1]
```diff
@@ -274,9 +274,18 @@ pd-server 相关配置项。
 
 ### `patrol-region-interval`
 
-+ 控制 replicaChecker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整
++ 控制 checker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整
 + 默认：10ms
 
+### `patrol-region-worker-count`
```
Comment (Oreoxmt) [inline pd-configuration-file.md:280]:
```suggestion
### `patrol-region-worker-count` <span class="version-mark">从 v8.5.0 版本开始引入</span>
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -274,9 +274,18 @@ pd-server 相关配置项。
...
+
+> **警告：**
+>
+> 在当前版本中，将该配置项设置大于 1 时开启并发，为实验特性，不建议在生产环境中使用。
```
Comment (Oreoxmt) [inline pd-configuration-file.md:284]:
```suggestion
> 将该配置项设置为大于 1 将启用并发检查。目前该功能为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
```

File: pd-configuration-file.md [hunk 3]
```diff
@@ -274,9 +274,18 @@ pd-server 相关配置项。
...
+
++ 控制 checker 检查 Region 健康状态时，创建 operator 的并发数，通常状况不需要调整
```
Comment (Oreoxmt) [inline pd-configuration-file.md:286]:
```suggestion
+ 控制 checker 检查 Region 健康状态时，创建 operator 的并发数。通常情况下，无需调整此配置项。
```

File: pd-control.md [hunk 4]
```diff
@@ -312,16 +312,26 @@ config show cluster-version
...
-    设置 replicaChecker 的运行频率为 10 毫秒：
+    设置 checker 的运行频率为 10 毫秒：
 
     {{< copyable "" >}}
 
     ```bash
     config set patrol-region-interval 10ms
     ```
 
+- `patrol-region-worker-count` 控制 checker 检查 Region 健康状态时，创建 operator 的并发数，通常状况不需要调整。在当前版本中，将该配置项设置大于 1 时开启并发，为实验特性，不建议在生产环境中使用。
```
Comment (Oreoxmt) [inline pd-control.md:325]:
```suggestion
- `patrol-region-worker-count` 控制 checker 检查 Region 健康状态时，创建 operator 的并发数。通常情况下，无需调整此配置项。将该配置项设置为大于 1 将启用并发检查。目前该功能为实验特性，不建议在生产环境中使用。
```

File: pd-control.md [hunk 5]
```diff
@@ -312,16 +312,26 @@ config show cluster-version
...
+
+    设置 checker 的并发数为2：
```
Comment (Oreoxmt) [inline pd-control.md:327]:
```suggestion
    设置 checker 的并发数为 2：
```

File: pd-control.md [hunk 6]
```diff
@@ -312,16 +312,26 @@ config show cluster-version
...
+
+    {{< copyable "" >}}
+
```
Comment (Oreoxmt) [inline pd-control.md:330]:
```suggestion
```
This copy syntax is no longer needed for the current docs website. Please remove it from this doc.

File: pd-configuration-file.md (hunk 2 above)
Comment (lhy1024) [reply pd-configuration-file.md:284]: Do we need to link it to pd?

Comment (Oreoxmt) [reply pd-configuration-file.md:284]: We can update the link to [issue](https://github.com/tikv/pd/issues) if we want users to report bugs in this repository

**Comment by lhy1024** [comment]
PTAL @Oreoxmt

**Review by Oreoxmt — APPROVED** [review]

File: pd-configuration-file.md [hunk 7]
```diff
@@ -274,9 +274,18 @@ pd-server 相关配置项。
...
++ 控制 checker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整
 + 默认：10ms
 
+### `patrol-region-worker-count` <span class="version-mark">从 v8.5.0 版本开始引入</span>
+
+> **警告：**
+>
+> 将该配置项设置为大于 1 将启用并发检查。目前该功能为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/tikv/pd/issues)反馈。
+
++ 控制 checker 检查 Region 健康状态时，创建 operator 的并发数。通常情况下，无需调整此配置项。
```
Comment (qiancai) [inline pd-configuration-file.md:286]: 配置项名字里用的是 worker，描述里用的 operator 需要统一下吗，或者可以改成下面这样吗

```suggestion
+ 控制 checker 在检查 Region 健康状态时的并发线程数量。通常情况下，无需调整此配置项。
```

Comment (qiancai) [inline pd-configuration-file.md:286]:
```suggestion
+ 控制 checker 检查 Region 健康状态时，创建 [operator](/glossary.md#operator) 的并发数。通常情况下，无需调整此配置项。
```

File: pd-control.md [hunk 8]
```diff
@@ -312,16 +312,24 @@ config show cluster-version
...
-    设置 replicaChecker 的运行频率为 10 毫秒：
+    设置 checker 的运行频率为 10 毫秒：
 
     {{< copyable "" >}}
 
     ```bash
     config set patrol-region-interval 10ms
     ```
 
+- `patrol-region-worker-count` 控制 checker 检查 Region 健康状态时，创建 operator 的并发数。通常情况下，无需调整此配置项。将该配置项设置为大于 1 将启用并发检查。目前该功能为实验特性，不建议在生产环境中使用。
```
Comment (qiancai) [inline pd-control.md:325]:
```suggestion
- `patrol-region-worker-count` 控制 checker 检查 Region 健康状态时，创建 [operator](/glossary.md#operator) 的并发数。通常情况下，无需调整此配置项。将该配置项设置为大于 1 将启用并发检查。目前该功能为实验特性，不建议在生产环境中使用。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**